From f367b73852f60d196bd5856d818730e08ca14270 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 17 Aug 2016 16:48:07 +0200
Subject: [PATCH] dracut.sh: document --hostonly-i18n and --no-hostonly-i18n

show description in --help

https://bugzilla.redhat.com/show_bug.cgi?id=1266448
(cherry picked from commit 91896ab68e26b9f9191f4742dc8ad9c5025eac7f)
---
 dracut.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/dracut.sh b/dracut.sh
index 36607cf..40ca08f 100755
--- a/dracut.sh
+++ b/dracut.sh
@@ -151,6 +151,9 @@ Creates initial ramdisk images for preloading modules
                         in the initramfs
   --no-hostonly-cmdline Do not store kernel command line arguments needed
                         in the initramfs
+  --hostonly-i18n       Install only needed keyboard and font files according
+                        to the host configuration (default).
+  --no-hostonly-i18n    Install all keyboard and font files available.
   --persistent-policy [POLICY]
                         Use [POLICY] to address disks and partitions.
                         POLICY can be any directory name found in /dev/disk.
